thread 'main' panicked at 'assertion failed: thread::spawn(|| i8::MIN.abs()).join().is_err()', ui/numbers-arithmetic/int-abs-overflow.rs:9:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
